### PR TITLE
Classless update

### LINKED ISSUE #13: 修饰classless样式
> 3. nav的ul换成两行、图片变为两行、aside消失是此css规定的行为。可以修改。
>  到时候要把ul换成html5规定的menu元素，aside弄大一点，在页面较窄的时候并到文章中……要改的东西还多着呢。

按照 https://github.com/wyast/wyast.github.io/issues/9#issuecomment-1938534042 的想法修改。

### PR DESCRIPTION
**Pull request by liuyifan22**
The version including side navigator, we may merge this into our main branch as our new basis. "Anchor" branch may be deleted.

### CODE REVIEW

File: style/classless.css [hunk 1]
```diff
@@ -10,212 +10,436 @@ Table of Contents:
...
+
+/* grid style for content elements */
+article,
+header,
+footer,
+article > section {
+  display: grid;
+  grid-template-columns: 2fr min(var(--width), calc(100% - 96px)) 5fr;
+  column-gap: 48px;
+}
```
Comment (RebelPotato) [inline style/classless.css]: “塌陷”的样式来自这里。使用css样式表，将左右间距设为2:5，可随意修改。
如果觉得难看，可以随时改回1:1。
```suggestion
/* grid style for content elements */
article,header,footer,article > section {
  display: grid;
  grid-template-columns: 1fr min(var(--width), calc(100% - 96px)) 1fr;
  column-gap: 48px;
}
```

**Review by RebelPotato — CHANGES_REQUESTED** [review]

File: style/classless.css (hunk 1 above)
Comment (RebelPotato) [reply style/classless.css]: 已经改正。

**Review by RebelPotato — APPROVED** [review]
🆗

**Review by An-314 — APPROVED** [review]

**Review by liuyifan22 — COMMENTED** [review]
现在的目录更加好看了：随着页面滚动而新出现的标题被加粗，美观清晰。辛苦了！我现在执行merge

**Comment by liuyifan22** [comment]
Merged and deleted "classless-update"

**Pull request merged**